packages/docs/site/docs/01-start-here/03-repos-overview.md

```diff
@@ -0,0 +1,79 @@
+# Understanding the Playground Repositories
+There are a couple of different repos that serve different purposes in the WordPress Playground project. Here's a quick overview of each:
+
+## [WordPress Playground](https://github.com/WordPress/wordpress-playground/)
+This is the main repository for the WordPress Playground project. It contains the core codebase for the project, project boards and issues, and discussions around ongoing project improvements and maintenance.
+
+The repo is there to guide the core contributors to the Playground identify issues, feature requests, and other improvements that need to be made to the project.
+
+Opening issues in the WordPress Playground repository for the "online" version of the Playground, IE: playground.wordpress.net, is a great way to get visibility on a problem or feature request. But there may be a better suited repo below to open the issue, and we'll discuss that shortly.
+
+Searching for issues is a great way to find a fix or workaround, or to see if a feature request has already been made. If you find an issue that matches your problem, you can add a comment to the issue to provide more information or to show that you are also experiencing the same problem.
+
+Explore the WordPress Playground [repository](https://github.com/WordPress/wordpress-playground/)
+
+---
+
+## [Playground Tools]((https://github.com/WordPress/playground-tools/))
+This repository contains a collection of tools and utilities that are used to manage and maintain the WordPress Playground project. The tools include scripts for building and deploying the project, as well as utilities for managing the documentation and other project assets.
+
+There are also advanced administration features, integrations such as the Playground Plugin, Blueprint Builder, and Interactive Code Block that live in the Playground Tools repository.
+
+This repository is also the best place to search for issues related to `wp-now` or the `VS Code Extension` for WordPress Playground. If the request you are looking for is not specific to the online version of the Playground, it may be in the Playground Tools repository.
+
+#### Tools that exist in the Playground Tools repository:
+- [Blueprint Builder](https://playground.wordpress.net/builder/builder.html) - a tool to create a view Blueprints side-by-side
```

Comment:
Nitpick: This one is actually in the main Playground repo 😅 I'd love to move it over, just didn't get to it yet

Reply:
But the playground-tools repo is still cool for opening issues

Reply:
So technically the Blueprint Builder is an online interface. The way I've been leaning towards the WordPress Playground repo was for "Playground online" - AKA playground.wordpress.net

Feel free to re-word or I can take a look at it later or tomorrow. Gonna switch gears for a little bit, thanks for checking in today!

Reply:
the Blueprint Builder is in the Playground repo... it's an online tool. That makes sense.

Will update...